### PR TITLE
Added migration warning for moving from version 1.68.0 to 1.69.0+

### PR DESCRIPTION
**Pull request by targeter21**

### CODE REVIEW

**Comment by derickfernando** [comment]
+0

File: README.md [hunk 1]
```diff
@@ -18,8 +18,9 @@
 
 # Sonatype Nexus IQ Server Docker: sonatype/nexus-iq-server
 
-A Dockerfile for Sonatype Nexus IQ Server, based on [Red Hat Universal Base Image](https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image).
+A Dockerfile for Sonatype Nexus IQ Server, based on [RedHat Universal Base Image](https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image).
```
Comment (whyjustin) [inline README.md]: Red Hat is two words.

File: README.md [hunk 2]
```diff
@@ -30,6 +31,14 @@ A Dockerfile for Sonatype Nexus IQ Server, based on [Red Hat Universal Base Imag
 * [Chef Solo for Runtime and Application](chef-solo-for-runtime-and-application)
 * [Project License](#project-license)
 
+## Migration
+### Upgrading from Version 68 or Earlier to Version 69 or Later
+Version 1.69.0 of the Docker image changed the base image from CentOS to RedHat UBI (Universal Base Image). As a result, the UID of the `nexus` user has changed. If you use this image with a persistent data volume you will need to run the following for your server to start up successfully:
```
Comment (whyjustin) [inline README.md]: Lines should be < 120 chars

Comment (whyjustin) [inline README.md]: `Red Hat`

**Review by whyjustin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (targeter21) [reply README.md]: Changed.

File: README.md (hunk 2 above)
Comment (targeter21) [reply README.md]: Changed.

Comment (targeter21) [reply README.md]: Fixed.

**Review by bigspotteddog — APPROVED** [review]
+1 Looks good to me!

**Pull request merged**